src/views/perms/AssetPermission/AssetPermissionDetail/AssetPermissionUser.vue

```diff
@@ -103,7 +102,8 @@ export default {
         onAddSuccess: (objects, that) => {
           this.$log.debug('Select value', that.select2.value)
           that.iHasObjects = [...that.iHasObjects, ...objects]
-          this.$store.commit('common/reload')
+          that.$refs.select2.clearSelected()
+          this.$refs.ListTable.reloadTable()
         }
       },
       groupRelationConfig: {
```

Comment:
This code does not have identified irregularities, potential issues, or optimization suggestions at this checkpoint of time.